db/migrate/058_name_unnamed_redactions.rb

```diff
@@ -0,0 +1,19 @@
+class NameUnnamedRedactions < ActiveRecord::Migration[5.0]
+  def self.up
+    Redaction.find_each do |redaction|
+        if redaction.title.empty?
+            redaction.title = redaction.id.to_s
+            redaction.save!
+        end
+    end
+  end
+
+  def self.down
+    Redaction.find_each do |redaction|
```

Comment:
There's no need to change the titles back here, but you do need to remove the constraint with `change_column` here.